src/ducks/balance/components/KonnectorIcon.jsx

```diff
@@ -23,10 +18,13 @@ class KonnectorIcon extends React.PureComponent {
   }
 
   render() {
+    const { className } = this.props
+
     return (
-      <span style={style}>
-        <UIAppIcon fetchIcon={this.fetchIcon} />
-      </span>
+      <UIAppIcon
+        fetchIcon={this.fetchIcon}
+        className={className ? className : styles.KonnectorIcon_default}
```

Comment:
We really want to totally override default style as soon as we pass a `className`?

Reply:
I don't think so. If urgent, put a comment and an issue :/